build_runner/lib/src/server/graph_viz.js

```diff
@@ -0,0 +1,90 @@
+window.$build = {}
```

Comment:
copyright notice on this and other new files

Reply:
done